packages/client/src/runtime/utils/decimalJsLike.ts

```diff
@@ -8,6 +8,7 @@ export interface DecimalJsLike {
   d: number[]
   e: number
   s: number
+  toFixed(): string
```

Comment:
This PR changes decimal-js-like definition to add `toFixed` method. I don't think it is a breaking change, since this interface is hack to accept `Decimal.JS` objects of different versions and they all contain this method.

Reply:
I agree this is fine